packages/cdk8s-cli/src/cli/index.ts

```diff
@@ -1,15 +1,32 @@
+import * as colors from 'colors';
 import * as yargs from 'yargs';
+import { upgradeAvailable } from '../upgrades';
 
-const args = yargs
-  .commandDir('cmds')
-  .recommendCommands()
-  .wrap(yargs.terminalWidth())
-  .showHelpOnFail(false)
-  .env('CDK8S')
-  .epilogue('Options can be specified via environment variables with the "CDK8S_" prefix (e.g. "CDK8S_OUTPUT")')
-  .help()
-  .argv;
-
-if (args._.length === 0) {
-  yargs.showHelp();
+async function main() {
+  const versions = upgradeAvailable();
+  if (versions) {
+    console.error('------------------------------------------------------------------------------------------------');
+    console.error(colors.yellow(`A new version ${versions.latest} of cdk8s-cli is available (current ${versions.current}).`));
+    console.error(colors.yellow('Run "npm install -g cdk8s-cli" to install the latest version on your system.'));
```

Comment:
Would it be useful here to also give the `brew` and `yarn` upgrade commands?

Reply:
Maybe we just link to the docs?

Reply:
Yes

Reply:
Done